Karamelfile

```diff
@@ -2,6 +2,7 @@ dependencies:
   - recipe: flink::install
     global:  
       - kagent::install
+      - hops::install
   - recipe: flink::taskmanager
```

Comment:
Are the `flink::taskmanager` and `flink::jobmanager` recipes still valid?

Reply:
No, but they template some options for flink-conf.yml I think, I will try to remove them in any case.